### ISSUE TITLE
pass gen not working

### ISSUE THREAD

**Issue by d3vv3** [seqizz/ulauncher-password-store#6]
When I type `pass gen` it says "Loading...", press `space` and it "Please enter a password length (by default it's 16)". Then I enter `16` and it says "Loading..." and won't do anything. Wouldn't work even with 1 as length.

Hope I described the issue well.

**Comment by seqizz** [comment]
Hi, do you have pwgen command available on your system? If not, that means I've failed to fail. If you have the command in your $PATH, could you give me the location of it via `which pwgen`.

**Comment by d3vv3** [comment]
Yes, I have both, pwgen and pwgen for Python. The output of `which pwgen` is "/usr/local/bin/pwgen"

**Comment by seqizz** [comment]
Please update and re-check. I guess I found the issue :+1:

**Comment by d3vv3** [comment]
Still the same issue... Maybe re-open issue?

**Comment by seqizz** [comment]
Then yes. My theories are done at this point. Could you open a terminal, kill the ulauncher process and re-launch manually to see the output and try generating a pass again_ Maybe we can see the fail is logged there:

`pkill -3 ulauncher && ulauncher`

**Comment by d3vv3** [comment]
Yes, it is definately logged there, there you have it, thanks!

[log.txt](https://github.com/seqizz/ulauncher-password-store/files/2556995/log.txt)

**Comment by seqizz** [comment]
Hmm, is your pwgen capable of correctly running? Because it seems like it's not a binary, but python3 script.
Can you try running `/usr/local/bin/pwgen -1 -c -n -y 16 8`

**Comment by d3vv3** [comment]
Yes, here you go:
`$ /usr/local/bin/pwgen -1 -c -n -y 16`
output: "A7BMGinNoZ}mXvK5"


When running `/usr/local/bin/pwgen -1 -c -n -y 16 8` it says: 

usage: pwgen [-h] [--capitalize] [--no-capitalize] [--numerals]
             [--no-numerals] [--symbols] [--ambiguous] [-1]
             [pw_length]
pwgen: error: unrecognized arguments: 8

**Comment by seqizz** [comment]
Then I'll need to verify if the pwgen is really binary or not too. All our pwgen commands are sent with `8` at the end, because I want more then 1 passwords generated for user to select. In my binary (version 2.07, which comes from my distro - debian) here is the result:
```
~ ~> pwgen -1 -c -n -y 16 8
kaiPh5eec\ood2th
xi3ahPhom&oReipi
eeJeo>quee8xaigh
oMaru-oquae2ohs:
jep;ae5dah:w@ooB
oow>ieTh(ae^gh1T
ahCh6ong_u5nooji
oong7aca~u,Z.ieX

```

**Comment by d3vv3** [comment]
Fixed! I had the pip installed pwgen instead of the binary! Works now, sorry for the inconvenience...

**Comment by seqizz** [comment]
Glad to hear, I will keep this issue open to remind me to write a test case about the pwgen and see if it is producing what we expect.

**Comment by d3vv3** [comment]
Actually, just tried to use pass gen, and it wont copy the chosen one to the clip... You can see the error in the last line, where it shows the chosen generated password.
[log.txt](https://github.com/seqizz/ulauncher-password-store/files/2559484/log.txt)

**Comment by seqizz** [comment]
That was another fail. Thanks for the report. Should be fixed with the last update.

**Comment by d3vv3** [comment]
Yes, it's fixed, thanks!
